node/node.go

```
@@ -13,6 +13,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/rs/cors"
 
+	"github.com/cometbft/cometbft/blocksync"
```

golangci-lint (1.64.8)

ST1019: package "github.com/cometbft/cometbft/blocksync" is being imported more than once

(stylecheck)